gnat2goto/driver/arrays.adb

```diff
@@ -762,36 +762,17 @@ package body Arrays is
                      Overflow_Check  => False,
                      I_Type          => Get_Type (Idx_Irep),
                      Range_Check     => False);
-      Result_Type : Irep;
-      Pointer_Type : constant Irep := New_Irep (I_Pointer_Type);
+
```

Comment:
Not part of this PR, but I'm wondering why we even need the `Base_Type` here? Shouldn't it be sufficient to have an `Irep` type?

Reply:
Removed.